Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/application_Form_tests.cy.js

```diff
@@ -0,0 +1,95 @@
+import '../../support/commands';
+import projectTaskList from "../../pages/projectTaskList";
+const applicationFormTaskList = require('../../pages/applicationFormTaskList');
+
+describe('Test Application Form for Voluntary and Form a MAT', { tags: ['@dev', '@stage'] }, () => {
+
+    beforeEach(() => {
+        projectTaskList.getHomePage();
+    });
+
+   it('Should filter and select the project, then verify details of Voluntary', () => {
+
+    // Step 1: Filter search
+    applicationFormTaskList.filterSearch('Fahads Cypress Trust');
+
+    // Step 2: Verify the title and route
+    applicationFormTaskList.checkTitle('Fahads Cypress Trust');
+    applicationFormTaskList.checkRoute('Voluntary conversion');
+
+    // Step 3: Select the first search result
+     applicationFormTaskList.selectFirstSearchResult();
+
+    // Step 4: Click on school application form
+    applicationFormTaskList.clickSchoolApplicationForm();
+
+    //Step 4.1: Check URL contains to contain school-application-form
+    applicationFormTaskList.checkUrlContainsSchoolApplicationForm();
+
+    //Step 4.2: Check the route 
+    applicationFormTaskList.checkRouteAppForm('Voluntary conversion');
+
+    // Step 5: Check the table contents
+    const expectedContents = [
+      'Overview',
+      'About the conversion',
+      'Further information',
+      'Finances',
+      'Future pupil numbers',
+      'Land and buildings',
+      'Pre-opening support grant',
+      'Consultation',
+      'Declaration'
+    ];
+    applicationFormTaskList.checkTableContents(expectedContents);
+
+    // Step 6: Check Overview1_value
+    applicationFormTaskList.checkElementText('[test-id="Overview1_value"]', 'PLYMOUTH CAST with Plymstock School');
```

Comment:
Just my personal preference

But I like the data-testid to be hidden in the function, rather than a generic method that passes the id in

e.g.

applicationFormTaskList.hasOverview("Some value")

Instead of

checkElementText("id", "This is my text")

It's consistent then with the other methods, e.g.

applicationFormTaskList.filterSearch("Fahads Cypress Trust")